tools/build/prepare_python.py

```diff
@@ -80,11 +93,21 @@ def prepare_for_platform(platform: str, arch: str,
 	shutil.unpack_archive(src_dir / pathlib.Path(config.source_url).name, extract_dir = src_dir)
 
 	src = src_dir / 'python'
+	src_lib_path = src / config.so_path
+	lib_filename = pathlib.Path(config.so_path).name
+
+	if platform == 'macos':
+		# Rename the library id (which we depend on) to be in @rpath.
+		# (it defaults to /install/lib/)
+		subprocess.run(['install_name_tool', '-id', f'@rpath/{lib_filename}', src_lib_path], check=True)
```

Comment:
This should maybe be performed on the copy instead, but that doesn't work because the copy isn't what is being linked, rendering the id change useless, and the final binary links to `/install/lib/`.